docs/07-AdvancedUsage.md

```diff
@@ -67,31 +67,29 @@ Also you can define `_failed` method in Cest class which will be called if test
 
 ## Dependency Injection
 
-Codeception supports simple dependency injection for Cest and \Codeception\TestCase\Test classes. It means that you can specify which classes you need as parameters of special `_inject()` method, and Codeception will automatically create respective objects and invoke this method, passing all dependencies as arguments. This can be useful when working with Helpers, for example:
+Codeception supports simple dependency injection for Cest and \Codeception\TestCase\Test classes. It means that you can specify which classes you need as parameters of special `_inject()` method, and Codeception will automatically create respective objects and invoke this method, passing all dependencies as arguments. This may be useful when working with Helpers, for example:
```

Comment:
as parameters of **the** special `_inject()` method

automatically create **the** respective objects